website/docs/api/commands/cli.composite.md

```diff
@@ -117,6 +123,12 @@ available on the Ceramic Node that yor DApp connects to. You can find a detailed
 guide on Composites' deployment
 [here](https://developers.ceramic.network/docs/composedb/guides/data-modeling/composites#deploying-composites)
 
+If updating your composite to add additional query fields, do not run this command.
```

Comment:
again, I think we need to be clearer with our language here about what exactly the problematic scenarios are and what the proper recourse is.

Reply:
Did the language here get updated?  I like the language you used for the `create` command above (mod one small suggestion), can we re-use that?